### DISCUSSION TITLE
Señores/as!!! ya esta listo el proyecto con Angular

### DISCUSSION THREAD

**Discussion by lionel1014** [pgCatamarca2021fs/aula1-grupoc-team-1c#24] [Announcements]
Cree la aplicación mediante ng new ejemplo y luego acomode los demás archivos del mismo... cualquier cosa pueden consultar por este foro y con gusto les respondo todas sus dudas.
Saludos! 🐱‍👤

**Comment by luisgm21** [comment]
Creo que voy a crear una rama para mi xD y ahi voy a programar unas cosas por separado y despues veremos lo del merge xD, una duda ya hiciste el header o el footer ? sino habria que ver que queda para que programe xdxdx

**Comment by lionel1014** [comment]
Hola Luissss estaba viendo un lindo navbar, ademas implemente el routing pero no lo subi si podes diseña un footer y un header. Y ademas si queres cuando puedas te muestro como va quedando el proyecto

**Reply by luisgm21** [reply]
ok nic , bien igual no esperes algo hermoso jajajaa soy mas del back

**Reply by luisgm21** [reply]
tengo una compu nueva, asi que si no es problema que version de node tenes y de angular asi tengamos la mismo y no haya incompatibilidad xd

**Reply by lionel1014** [reply]
No tendría que haber problemas si clonas la version de la rama principal. ya que en el package.json le dice que versión de dependencias necesitara el proyecto y lo bajara automaticamente

**Reply by luisgm21** [reply]
Perfecto 😎
